ports/chromium-base/portfile.cmake

```diff
@@ -51,6 +51,15 @@ checkout_in_path(
     "${CHROMIUM_GIT}/third_party/modp_b64" 
     "509f005fa65e652dc4a6f636da6fa1002b6dce16")
 
+if(VCPKG_CMAKE_SYSTEM_NAME STREQUAL Linux)
+    vcpkg_download_distfile(ARCHIVE
```

Comment:
Would it be possible to avoid this by setting the `use_sysroot` arg to false?

Reply:
OK，我试一下

Reply:
> Would it be possible to avoid this by setting the `use_sysroot` arg to false?

the archive looks like contain a lot of third party library, does vcpkg provide `apt install` function？By the way, docker image may be provide a minimal environment.

Reply:
@iHuahua, I checked out your branch and managed to get the port to build on an Ubuntu 18.04 VM. Could you try removing the `chrome-linux-sysroot` download and adding these options:
```
use_sysroot=false
use_glib=false
```
A recent Clang/LLVM is also required:
```
sudo apt install clang-10 llvm-10
sudo ln -sf /usr/bin/clang++-10 /usr/bin/clang++
sudo ln -sf /usr/bin/clang-10 /usr/bin/clang
sudo ln -sf /usr/bin/llvm-ar-10 /usr/bin/llvm-ar
```

Reply:
@vejmartin use_glib=false. You are a genius, it workes, I have committed the change.
But I got an other link issue with static library when I link base to an executable, it works with dynamic library, do you have encountered？